src/sentry/tasks/merge.py

```diff
@@ -257,11 +257,16 @@ def merge_objects(models, group, new_group, limit=1000,
                             )
                     elif model == GroupTagValue:
                         with transaction.atomic(using=router.db_for_write(model)):
```

Comment:
It is, so not entirely sure. If you wanna test this behavior, feel free. :) I'm not sure if it's actually needed or not.